CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+# Changelog
+
```

Comment:
You've included a package.json update in this PR, please add the change log here